### ISSUE TITLE
Are modules supported?

### ISSUE THREAD

**Issue by shish** [exaloop/codon#159]
I am having problems with modules, and I don't see them mentioned anywhere in the documentation, so I don't know what is _supposed_ to work :( It seems like all of the examples are single .py or .codon files, is that all that's supported?

Specifically: 
```
$ git clone https://github.com/shish/rosettaboy
$ cd rosettaboy/py

$ python3 main.py --help
usage: main.py [-h] [--info] [-c] [-g] [-r] [-H] [-S] [-t] [-f N] [-p N] rom

positional arguments:
  rom

options:
  -h, --help         show this help message and exit
  --info             Show ROM metadata
  -c, --debug-cpu
  -g, --debug-gpu
  -r, --debug-ram
  -H, --headless
  -S, --silent
  -t, --turbo
  -f N, --frames N   Exit after N frames
  -p N, --profile N  Exit after N seconds

$ codon run main.py --help
main.py:2:6-14: error: no module named 'src.main'
main.py:4:10-14: error: name 'main' is not defined
```

**Comment by markshannon** [comment]
The simplest of imports, just a file containing:
```
import json
```
fails.

**Comment by inumanag** [comment]
I'll take a look at this.

FYI large portions of Python's stdlib (such as `json`) are not yet implemented natively in Codon. If you want to use Python's version, you need to do `from python import json` for now.

**Comment by arshajii** [comment]
This should be fixed in the latest release (0.15.4); please let us know if you encounter any other issues with this.
